src/world/map.c

```diff
@@ -1638,15 +1638,15 @@ static money32 map_set_land_height(int flags, int x, int y, int height, int styl
 		return MONEY32_UNDEFINED;
 	}
 
-	if (height > 62) {
+	if (height > 270) {
```

Comment:
Can you add a comment stating this is a change from vanilla. Also don't sneak changes in on a unrelated fix